Compilation errors

\<root>/stinger/build/include/stinger_alg/streaming_connected_components.h:14:37: fatal error: stinger_net/stinger_alg.h: No such file or directory

Full build log: https://pastebin.com/UQJ9YAFM

--- a/lib/stinger_alg/CMakeLists.txt
+++ b/lib/stinger_alg/CMakeLists.txt
@@ -51,4 +51,4 @@ include_directories("${CMAKE_BINARY_DIR}/include/stinger_net")
 
 set_source_files_properties(${config} PROPERTIES GENERATED TRUE)
 add_library(stinger_alg ${sources} ${headers} ${config})
-target_link_libraries(stinger_alg stinger_core compat)
+target_link_libraries(stinger_alg stinger_core stinger_net stinger_utils compat)

